### DISCUSSION TITLE
Queue Stuck

### DISCUSSION THREAD

**Discussion by christopherpeers** [cyberbotics/wrestling#79] [General]
I believe there may be an issue with the queue/actions pipeline since its displaying that there are 6 people in queue with no actions currently running.

**Comment by omichel** [comment]
Right, I restarted the queue and it seems it is now running smoothly.

**Comment by omichel** [comment]
Well, it seems GitHub is currently having issues with actions: https://www.githubstatus.com

**Comment by kibergrad** [comment]
It seems that the Queue functionality is stuck again.

**Comment by omichel** [comment]
It's up again now.

**Reply by kibergrad** [reply]
Stuck Again.

**Reply by omichel** [reply]
Up again.
I am investigating what is causing those problems. It seems it doesn't come from GitHub actions since they seem to be working now.

**Comment by ImanolSAGA** [comment]
Stuck again :(

**Reply by omichel** [reply]
Oops, I just fixed it again.

**Reply by AmoghBhagwat** [reply]
Stuck again :)

**Reply by BenjaminDeleze** [reply]
Sorry, it should be fixed
